docs/example_workflow_explained.rst

```diff
@@ -0,0 +1,316 @@
+
+.. code:: ipython3 
+
+    watershed_raw = xr.open_mfdataset('../input/first_route*.nc')[['IRFroutedRunoff', 'dlayRunoff', 'reachID']].load()
+    watershed_raw['seg'] = watershed_raw.isel(time=0)['reachID'].astype(np.int32)
+    watershed_ref = xr.open_dataset('../input/nrni_reference_flows.nc').load().rename({'outlet':'site'})[['seg', 'seg_id', 'reference_flow']]
+    
+In order to select data for the basin of analysis from the larger watershed, we 
+need the topology of the larger watershed as well.
+
+.. code:: ipython3 
+
+    watershed_topo = xr.open_dataset('../topologies/watershed_topology_file_name.nc').load()
+    watershed_topo = watershed_topo.where(watershed_topo['hru'] < 1.79e7, drop=True)
+    
+    # Here we clean up a few naming conventions to get everything on the same page.
+    if 'hru_id2' in basin_topo:
+        basin_topo['hru'] = basin_topo['hru_id2']
+    if 'seg_id' in basin_topo:
+        basin_topo['seg'] = basin_topo['seg_id']
+
+
+
+
+Convert ``mizuroute`` formatting to ``bmorph`` formatting
+---------------------------------------------------------
+
+``mizuroute_utils`` is our utility script that will handle converting
+Mizuroute outputs to what we need for ``bmorph``. For more information
+on what ``mizuroute_utils`` does specifically and how to change its 
+parameters, check out ``data.rst``.
+
+Here we pull out coordinate data from the ovearching watershed
+for the specific basin we want to analyze.
+    
+.. code:: ipython3 
+    
+    basin_ref = watershed_ref.sel(site=[r for r in ref_sites])
+    
+    for site, seg in site_to_seg.items():
+        if site in basin_ref['site']:
+            basin_ref['seg'].loc[{'site': site}] = seg
+    
+Now we pass it off to ``mizuroute_to_blendmorph``, the primary utility 
+function for automating ``bmorph`` pre-procesing.
+    
+.. code:: ipython3 
+
+    basin_met_seg = mizutil.mizuroute_to_blendmorph(
+        basin_topo, watershed_raw.copy(), basin_ref, watershed_met, 
+        fill_method='r2').ffill(dim='seg')
+
+Apply ``bmorph`` bias correction
+--------------------------------
+
+We are almost to actually bias correcting! First we need to specify some parameters 
+for correction. Returning to these parameters can help fine tune your bias 
+corrections to the basin you are analyzing.
+
+In this notebook, all four variations of ``bmorph`` are demonstrated: 
+IBC_U, IBC_C, SCBC_U, and SCBC_C, as described in ``bias_correction.rst``.
+
+The ``train_window`` is what we will use to train the bias correction
+model. This is the time range that is representative of the
+basin's expected behavior that ``bmorph`` should strive to mirror.
+
+The ``bmorph_window`` is when ``bmorph`` should be applied to the series for
+bias correction.
+
+Lastly the ``reference_window`` is when the reference flows should be used to 
+smooth the bias corrected flows. This is recommended to be set as equivalent to the
```

Comment:
I need to double check this, but I think `smooth the bias corrected flows` should be `correct the raw flows` here, right?

Reply:
`    # Apply the correction
    if n_smooth_long:
        nrni_mean = obs_ts[reference_window].mean()
        train_mean = train_ts[reference_window].mean()
        bmorph_corr_ts = bmorph.bmorph_correct(raw_ts, bmorph_ts, raw_ts_window,
                                               nrni_mean, train_mean,
                                               n_smooth_long)`
                                               
This is the only context that it is used, so I think I'm correct.